### ISSUE TITLE
Don't flag numeric properties on objects

### ISSUE THREAD

**Issue by danielnixon** [danielnixon/eslint-plugin-total-functions#15]
```
declare const grey: {
  50: '#fafafa';
  100: '#f5f5f5';
}

// Shouldn't be flagged.
const foo = grey[200];
```

**Comment by danielnixon** [comment]
Symbols are probably similarly broken
